README.md

```diff
@@ -705,8 +667,59 @@ normal you can usually fix it by restarting the two scripts and running QEMU
+[here](https://www.zephyrproject.org/doc/subsystems/networking/qemu_setup.html)
 
 ## Networking with BLE
-It is possible to use IP networking over BLE using 6lowpan. This is explained
-in a dedicated [document](./docs/6lowpan-ble.md)
+It is possible to use IP networking over BLE using 6LowPAN. This is explained
+in a dedicated [document](./docs/6lowpan-ble.md).
+
+
+## Contributing
+
+If you want to contribute code to the ZJS project, first you need to fork the
+project. The next step is to send a pull request (PR) for review to the ZJS
+repository. The PR will be reviewed by the project team members. You need at
+least two plus-ones (+1) , "Look Good To Me (LGTM)" or other positive signals
+for the project members. Once you have gained the required signals the project
+maintainers will merge the PR.
+
+### Travis and local sanity checking
+
+We run a series of tests on each pull request and merged commit using Travis.
+This relies on a script in the repo called `trlite`. One easy way to run these
+tests on your local `$ZJS_BASE` git tree is to use `make check` or
+`make quickcheck` for a faster subset of the tests. These run with your code as
+it stands in your tree. This will not catch a problem like you failing to
+add a new file to your commit.
+
+For a slightly safer sanity check, which might catch that kind of problem, you
+can run `trlite` directly or `trlite linux` for the "quick subset". This will
+clone a second copy of your git tree into a .trlite subdirectory, apply changes
+that `git diff` knows about, and run the build tests. Another option `trlite -j`
+will cause it to run four threads of tests to speed up execution; these will
+use four directories named `.trlite[1-4]`. If there is a test failure, the
+affected `.trlite*` trees are left in place so that you can investigate.
+
+## Repo Organization
+
+### File Descriptions
+* `zjs-env.sh` - Source this file to set environment variables and path to be
+able to use tools from ```scripts/``` anywhere.
+* `prj.conf` - The main configuration file for a Zephyr application; overrides
+settings from a defconfig file in the Zephyr tree. In the ZJS builds, we
+assemble the prj.conf file at build time from other fragments.
+* `prj.mdef` - Another configuration file for a Zephyr application; we use it to
+configure the heap size available to the ZJS API.
+
+### Subdirectories
+- `arc/` - Contains sensor subsystem code for ARC side of the Arduino 101.
+- `deps/` - Contains dependency repos and scripts for working with them.
+- `docs/` - Documentation in Markdown format (use API.md as index).
+- `outdir/` - Directory generated by build, can be safely removed.
+- `samples/` - Sample JavaScript files that can be built with make JS=<path>.
+- `scripts/` - Subdirectory containing tools useful during development.
+- `src/` - JS API bindings for JerryScript written directly on top of Zephyr.
+- `tests/` - JavaScript unit tests (incomplete).
+
+
+[Web IDE]: https://01org.github.io/zephyrjs-ide
```

Comment:
What did you intend this bit to do? It doesn't seem to show up... other than that +1 with my little fix.